gtk3/lib/gtk3/toggle-button.rb

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2015  Ruby-GNOME2 Project Team
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+module Gtk
+  class ToggleButton
+    alias_method :initialize_raw, :initialize
+    def initialize(options={})
+      label    = options[:label]
+      mnemonic = options[:mnemonic]
```

Comment:
In my understanding, `:use_underline` means that "I use underline in label to indicate mnemonic" not "I expect that label is displayed with underline".